models/user.test.js

```diff
@@ -32,3 +32,15 @@ describe('validate uniqueness', () => {
     );
   });
 });
+
+describe('validate presence', () => {
+  test('username is required', async () => {
+    let invalidUser = new User({ password: '123' });
+    await expect(invalidUser.save()).rejects.toThrow(ValidationError);
+  });
+
+  test('username is required', async () => {
+    let invalidUser = new User({ username: 'sabrina2' });
+    await expect(invalidUser.save()).rejects.toThrow(ValidationError);
+  });
+});
```

Comment:
Something weird going on here -- it passes even when the User model does not have `required: true` :neckbeard: ❓